src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import Button from './Button';
+
+const ButtonPanel = ({ handleClick }) => (
```

Comment:
- [ ] It would be better to implement the handle click function instead of lifting the function from the parent component to the child component as props
![Screenshot from 2021-10-18 16-24-57](https://user-images.githubusercontent.com/46388113/137770891-92cc8085-7b7d-44d1-8432-61c78f0f18e7.png)